tasks/build/package_scripts/pre_install.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+
+if command -v systemctl >/dev/null && systemctl is-active kibana.service >/dev/null; then
+    systemctl --no-reload stop kibana.service
+fi
+if [ -x /etc/init.d/kibana ]; then
+    if command -v invoke-rc.d >/dev/null; then
+        invoke-rc.d kibana stop
+    elif command -v service >/dev/null; then
```

Comment:
It doesn't invoke the same stop, the script for that one is at `kibana/build/services/systemd/lib/systemd/system/kibana.service`.  Because sysv and systemd scripts are included and we don't know which one is used the is-active check makes sure we check both if needed.